### DISCUSSION TITLE
Lexus Vehicles

### DISCUSSION THREAD

**Discussion by dnldcs** [widewing/ha-toyota-na#24] [General]
I know Toyota is the parent company of Lexus, so I tried using my Lexus login. Good to go, everything seems to be working, except I don't see Remote Start & Lock/Unlock. Didn't make this an issue, since it is not designed to work with Lexus.
![Lexus 1](https://user-images.githubusercontent.com/83604825/152565306-1d057bf0-523b-408f-b745-9b8232fc41a1.JPG)

**Comment by widewing** [comment]
Glad to hear it works for Lexus as well!  The remote functions are currently exposed as HA service not as button or switch, so you won't see it in the entity list for now.

**Reply by dnldcs** [reply]
Completely right! I used developer tools and was able to do remote start and locks. Thanks!
